### ISSUE TITLE
Unread messages on the message list not displayed in bold (when using non-default system font)

### ISSUE THREAD

**Issue by Winger9999** [thunderbird/thunderbird-android#1865]
### Expected behavior
Tell us what should happen
Unread messages on the message list should be displayed using bold font. 

### Actual behavior
Tell us what happens instead
Unread messages on the message list are not shown using bold font. The font used is the same as for read messages 

### Steps to reproduce
1. Syncronise
2. Enter the message list view
3. Verify the font used for unread messages 

### Environment
K-9 Mail version:
5.200 RC2

Android version:
6.01

Account type (IMAP, POP3, WebDAV/Exchange):
IMAP

**Comment by cketti** [comment]
The first line in the message list (usually the subject) is displayed using bold text. If this doesn't work it's most likely an issue with your system font.

**Comment by Winger9999** [comment]
I did more research on this one and have the following comments. Yes, you are right, it has something to do with the HTC system font and recent release of K-9, but due to the fact that I am not a programmer I can only share my observations, maybe it can help to resolve that issue:

1. The issue appeared only in the 5.200 RC2 release and didn't appear in previous releases I used (5.010). I use K-9 for many years and customize fonts on all of my phones and so far didn't encounter that issue, this is the first time.
2. The fonts that I use are HTC Theme fonts delivered officially by HTC (Settings/Personalize/Font Style). The K-9 shows the bold font for the subject of the unread message only for the Default Font and doesn't show in bold the subject of the unread message for any of the other of HTC fonts used (didn't check all of them, checked 20 randomly picked). This means that it is possible that any HTC user that changed his/her system font in phone settings will encounter the same problem.

Is there any way you can help?

**Comment by cketti** [comment]
It's not an important issue for me. But maybe someone else wants to pick this up.

**Comment by dbachran** [comment]
Well, I just recognized this after K9 got updated this week to 5.201, previously using 5.010.

I am using a HTC One M9 (with "custom" Roboto font via HTC Theme, as I was used to this from my previous phone). Android version is 6.0.

A quick work-around for me was to enable background highlighting for read mails to even have a chance to see if/which mails are still unread.

Is there any way I could help debugging/analyzing this any further?

Regards, Daniel
